docs/app/documentation/component-docs/dropdown/examples/dropdown-icons-example.component.html

```diff
@@ -1,12 +1,39 @@
 <fd-popover>
     <fd-dropdown-control [glyph]="'filter'">Dropdown</fd-dropdown-control>
-    <fd-popover-body *ngIf="menu1 && menu1.length">
+    <fd-popover-body *ngIf="values && values.length">
         <fd-menu>
             <ul fd-menu-list>
-                <li *ngFor="let option of menu1">
-                    <a fd-menu-item [attr.href]="option.url">{{option.text}}</a>
+                <li *ngFor="let option of values">
+                    <fd-icon size="l" [glyph]="option.icon" class="fd-template-icon"></fd-icon>
```

Comment:
I think we should wait for `menu-addon` directive to be released. It's dedicated to be used with menu items.